docs/units/logarithmic_units.rst

```diff
@@ -137,6 +137,29 @@ appropriate equivalency::
     >>> V.to(u.ABmag, u.spectral_density(5500.*u.AA))  # doctest: +FLOAT_CMP
     <Magnitude [16.99023831, 21.10978201, 22.50053827] mag(AB)>
 
+This is particularly useful for converting magnitude into flux density.
+``V`` is currently in ST magnitudes, which is based on flux densities per
+unit wavelength::
+
+    >>> flam = V.to(u.erg/u.s/u.cm**2/u.AA)
+    >>> flam # doctest: +FLOAT_CMP
+    <Quantity [5.75439937e-16, 1.29473986e-17, 3.59649961e-18] erg / (Angstrom cm2 s)>
+
+To convert ``V`` to flux density per unit frequency (on which AB magnitudes
+are based), we again need the appropriate equivalency::
+
+    >>> fnu = V.to(u.erg/u.s/u.cm**2/u.Hz, u.spectral_density(5500*u.AA))
+    >>> fnu # doctest: +FLOAT_CMP
+    <Quantity [5.80636959e-27, 1.30643316e-28, 3.62898099e-29] erg / (cm2 Hz s)>
+
+We could have also used frequency for the equivalency::
+
+    >>> lam = 5500*u.AA
+    >>> nu = c.c.to('AA/s') / lam
```

Comment:
Calculating `nu` like this is confusing to me. If you already know `lam`, there is no need to calculate `nu` and then redo the same conversion. So it makes more sense if you hardcode the value of `nu` here and use it directly. To make it clear that this only makes sense if user already have `nu` anyway.

Reply:
Hard coding sounds good to me! 

```suggestion
    >>> nu = 5.45077196e+14 * u.Hz
```